src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -0,0 +1,124 @@
+package api.loans;
+
+import static api.support.matchers.LoanMatchers.hasLoanProperty;
+import static api.support.matchers.LoanMatchers.hasOpenStatus;
+import static api.support.matchers.LoanMatchers.hasStatus;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.net.MalformedURLException;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import api.support.APITests;
+import api.support.builders.DeclareItemLostRequestBuilder;
+import api.support.http.InventoryItemResource;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+public class DeclareLostAPITests extends APITests {
+
+  private InventoryItemResource item;
+  private IndividualResource user;
+  private JsonObject loanJson;
+
+  @Override
+  public void beforeEach()
+    throws MalformedURLException, InterruptedException, ExecutionException,
+    TimeoutException {
+    super.beforeEach();
```

Comment:
Personally I’d introduce white space to separate the exceptions from the method body.

Reply:
Agree.